client/src/components/Froms/CreateTest.js

```diff
@@ -122,6 +126,11 @@ export default function CreateTest(porps) {
   ]);
 
   const handleNext = () => {
+     if (!name && !marks) {
```

Comment:
update this condition in a way that it shows a message for the individual field as well. like if the teacher inputted only the test name but not the marks then this will show that "please provide test marks". Same for test name and same if both fields is empty.

Reply:
@mohit355  done